test_all.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
```

Comment:
It required that we fix some things everywhere at once to get useful output. Now that we've done that we definitely don't need this.